assignments/prototypes.js

```diff
@@ -44,33 +44,128 @@
   - Give persons the ability to eat edibles.
   - When eating an edible, it should be pushed into a "stomach" property which is an array.
   - Give persons the ability to poop.
-  - When pooping, the stomach should empty.
+  - When pooping, the stomach should empty. */
 
-  TASK 2
+  function Person (name,age) {
+    this.name = name;
+    this.age =age;
+    this.stomach = [];
+  }
+  Person.prototype.greet = function() {
+    return `Hi! My name is ${this.name}, I am ${this.age} years old`;
+  }
+  
+  Person.prototype.eat = function(food) {
+    this.stomach.push(food);
+  }
+  Person.prototype.poop = function(){
+    this.stomach = [];
+  }
+
+  // const moyo = new Person("Moyo", 12)
+  // console.log(moyo.greet());
+  // moyo.eat('pasta');
+  // console.log(moyo.stomach);
+  // moyo.poop()
+  // console.log(moyo.stomach)
+
+
+  /*TASK 2
 
   - Build a Car constructor that takes model name and make.
   - Give cars the ability to drive a distance.
   - By driving a car, the distance driven should be added to an "odometer" property.
   - Give cars the ability to crash.
   - A crashed car can't be driven any more. Attempts return a string "I crashed at x miles!", x being the miles in the odometer.
   - Give cars the ability to be repaired.
-  - A repaired car can be driven again.
+  - A repaired car can be driven again. */
+
+  function Car(modelName, make) {
+    this.model = modelName;
+    this.make = make;
+    this.odometer = 0;
+    this.isCrashed = false;
+  }
+  Car.prototype.drive = function(distance){
+    if(this.isCrashed === false){
+      this.odometer += distance;
```

Comment:
This doesn't work because you are  not returning anything if the first condition is met